website/ref/path.md

```diff
@@ -0,0 +1,10 @@
+<!-- toc -->
+
+# Introduction
+
+The `path:` module provides path evaluation functions.
```

Comment:
"Path evaluation functions" doesn't sound right, "evaluating a path" isn't a well-defined thing. "Path manipulation functions" would be better.